src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 4 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Default parameter set is 'Ping'
```powershell
Expected the value to have type [System.Net.NetworkInformation.PingOptions] or any of its subtypes, but got $null with type $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 41
41:             $result[0].Options | Should -BeOfType "System.Net.NetworkInformation.PingOptions"
```
Test-Connection.Ping.BufferSize Should -Be between 0 and 65500
```powershell
Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 152
152:             { Test-Connection $targetName -BufferSize 0 } | Should -Not Throw

```
Test-Connection.MTUSizeDetect.MTUSizeDetect works
```powershell
Expected the actual value to be greater than 0, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 217
217:             $result.MtuSize | Should -BeGreaterThan 0
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.1.0.4', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 235
235:             $result[0].DestinationAddress | Should -BeExactly $realAddress
```